auto-pr-script/conv_proj_version_to_branch

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
+
+require "set"
+
+versions = ARGV
+default_branch = `gh repo view --json defaultBranchRef --jq '.defaultBranchRef.name'`.strip
```

Comment:
IIRC, I put the logic to determine the proper branch into this script so that we can easily test the logic. It would be difficult to test the logic if this script fetches the default branch from GitHub API. How about getting the default branch as an argument?

Reply:
Ok, it makes sense. I will move this line into the action file and pass the default branch as argument.